bundles/org.openhab.binding.danfossairunit/src/main/java/org/openhab/binding/danfossairunit/internal/DisconnectedException.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.danfossairunit.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Represents a communication failure with the air unit when it has not been connected.
+ *
+ * @author Robert Bach - Initial contribution
+ */
+@NonNullByDefault
+public class DisconnectedException extends RuntimeException {
```

Comment:
This type of Exception isn't handled in any special way in the rest of the code. I would suggest using an IOException instead.

Reply:
This exception isn't used anywhere now, please delete this class